src/js/layout.js

```diff
@@ -15,9 +15,11 @@ const processClassName = (data, field) => {
     }
 
     // Now that the col- types were lifted, remove from the actual input field
-    for (let index = 0; index < classes.length; index++) {
-      const element = classes[index]
-      field.classList.remove(element)
+    if (field.classList) {
```

Comment:
what is `field`'s value when `classList` is not defined?

Reply:
I noticed this error only when I added the Custom Autocomplete field from the demo. In this case field.classList was undefined. I am not sure what that was

Reply:
Looks like the Star Rating also has this phenomenon